dev-lang/micropython/micropython-1.25.0.ebuild

```
@@ -0,0 +1,79 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Python implementation for microcontrollers"
+HOMEPAGE="https://micropython.org https://github.com/micropython/micropython"
+SRC_URI="https://micropython.org/resources/source/${P}.tar.xz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	dev-libs/libffi:=
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	default
+	cd "${S}/ports/unix" || die
```

```suggestion
	cd ports/unix || die
```

`${S}` is the current directory, right?

---

Yes, you're right. Using "${S}" isn't necessary here.